shipyard-controller/internal/controller/shipyardcontroller.go

```diff
@@ -239,7 +238,8 @@ func (sc *ShipyardController) handleSequenceTriggered(event apimodels.KeptnConte
 	sequence, err := GetTaskSequenceInStage(eventScope.Stage, taskSequenceName, shipyard)
 	if err != nil {
 		msg := fmt.Sprintf("Unable to start sequence %s: %v", taskSequenceName, err)
-		log.Error(msg)
+		// use 'warn' level here since a sequence not being present is not an error in the service
+		log.Warn(msg)
```

Comment:
```suggestion
		log.Warnf("Unable to start sequence %s: %v", taskSequenceName, err)
```

Reply:
The `msg` var is then also passed to the function that emits the finished event containing that message, so I would prefer to keep it this way